README.md

```diff
@@ -30,6 +30,16 @@ PHP Requirements
 * `PHP 5.3+`
```

Comment:
Should update this to 5.4 for short array syntax support.

Reply:
You are right, I've updated docs and added composer file restriction for php 5.4+, also added travis environment for php 5.4 but build fail with error:

  Problem 1
    - Installation request for satooshi/php-coveralls v2.0.0 -> satisfiable by satooshi/php-coveralls[v2.0.0].
    - satooshi/php-coveralls v2.0.0 requires php ^5.5 || ^7.0 -> your PHP version (5.4.45) does not satisfy that requirement.

I don't know if we are ok with supporting php version 5.5+?